### PR TITLE
Overmap Announcements Overhaul

### PR DESCRIPTION
**Pull request by Geevies**
* Tuned up a lot of overmap event announcements, they will now announce to the appropriate ship. For shuttles, a small ping and a readout will come from the sensors console instead of a big announcement.
* Fixed a lot of events targetting the wrong ship, such as the ion storm one.
* Temporarily disabled the gravity event, as it affects ships globally instead of per-ship.

https://github.com/Aurorastation/Aurora.3/assets/22774890/b7b80308-e944-4ac3-9d8c-02a3c65b69e1

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Review by FluffyGhoster — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/events/electrical_storm.dm [hunk 1]
```diff
@@ -65,9 +64,21 @@
...
 	for (var/zlevel in affecting_z)
 		if(zlevel in current_map.station_levels)
 			command_announcement.Announce("The [location_name()] has cleared the electrical storm. Please repair any electrical overloads.", "Electrical Storm Alert", zlevels = affecting_z)
 			break
+
+
+/datum/event/electrical_storm/overmap/announce_start()
+	if(affecting_shuttle)
+		send_sensor_message("Entering electrical storm.")
+		return
```
Comment (NonQueueingMatt) [inline code/modules/events/electrical_storm.dm:77]: Why not just do `Entering [name]` on the parent? You're using affecting_shuttle here anyway

**Review by NonQueueingMatt — DISMISSED** [review]

**Comment by FluffyGhoster** [comment]
Reopening per author request

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
this will need a testmerge, too, to make sure i got everything

File: code/modules/overmap/ships/computers/sensors.dm [hunk 2]
```diff
@@ -80,6 +80,11 @@
 	else if(sound_token)
 		QDEL_NULL(sound_token)
 
+/obj/machinery/computer/ship/sensors/proc/display_message(var/message)
+	if(NOT_FLAG(stat, NOPOWER))
```
Comment (Arrow768) [inline code/modules/overmap/ships/computers/sensors.dm:84]: You should check for INOPERABLE instead

**Review by Arrow768 — DISMISSED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by ondrej008** [comment]
When calling a distress signal, it uses the name of the Horizon instead of the current ship as the announcement.

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**